node/subsystem-util/src/nesting_sender.rs

```diff
@@ -0,0 +1,148 @@
+// Copyright 2022 Parity Technologies (UK) Ltd.
```

Comment:
nit: I'm going to be that guy:

```suggestion
// Copyright 2023 Parity Technologies (UK) Ltd.
```